edxval/api.py

```diff
@@ -977,8 +977,8 @@ def create_transcripts_xml(video_id, video_el, resource_fs, static_dir):
                 video_id=video_id,
                 language_code=language_code,
                 file_format=file_format,
-                resource_fs=resource_fs.delegate_fs(),
-                static_dir=combine(u'course', static_dir)  # File system should not start from /draft directory.
+                resource_fs=resource_fs,
```

Comment:
So `deletegate_fs()` a method for finding the parent directory?

Can you give me an example of what `resource_fs` might be in practice?

Reply:
ResourceFS is an object of like `SubFS(OSFS(u'/tmp/tmpSTifrk'), u'/MITx-7.28r_4-2019_Spring')` which is a class from the `fs` package.

Reply:
The `delegate_fs` function call strips off the outer layer so it's just `OSFS(u'/tmp/tmpSTifrk')`